clients/admin-ui/src/features/system/SystemFormTabs.tsx

```diff
@@ -165,10 +173,10 @@ const SystemFormTabs = ({
       content: (
         <>
           <Box px={6} mb={9}>
+            {/* ts-ignore */}
```

Comment:
```suggestion
```

I think we added think to get around linting issues. See if you can remove this without issue